src/Diagnostics.RuntimeHost/Startup.cs

```diff
@@ -145,20 +146,20 @@ public void ConfigureServices(IServiceCollection services)
                 services.AddControllers(options =>
                 {
                     options.Filters.Add<AllowAnonymousFilter>();
-                }).AddNewtonsoftJson(options =>
+                }).AddJsonOptions(options =>
                 {
-                    options.SerializerSettings.Formatting = Formatting.Indented;
+                    options.JsonSerializerOptions.WriteIndented = true;
                 });
             }
             else
             {
                 services.AddControllers().AddJsonOptions(options =>
                 {
                     options.JsonSerializerOptions.WriteIndented = true;
-                }).AddNewtonsoftJson(options =>
+                }).AddJsonOptions(options =>
```

Comment:
CompilerHost Startup.cs also has `.AddNewtonsoftJson()` method. Should we change that as well? https://github.com/Azure/Azure-AppServices-Diagnostics/blob/b31dd4f95cacf16e81035c2a168136ff386d7acd/src/Diagnostics.CompilerHost/Startup.cs#L155

Reply:
That is not required. We are just changing the serializer in the runtime host project.